ansible/clean.yml

```diff
@@ -35,7 +35,23 @@
   tasks:
```

Comment:
@rajat-soda have you added the `group_vars/gelato-ha.yml` to the vars_files list for the main cleaning tasking?

Reply:
Done